regparser/builder.py

```diff
@@ -94,7 +96,20 @@ def reg_tree(reg_str):
         if reg_str[:1] == '<':  # XML
             return reg_text.build_tree(reg_str)
         else:
-            return build_whole_regtree(reg_str)
+            raise ValueError("Building from text input is no longer "
+                             "supported")
+            # return build_whole_regtree(reg_str)
+
+    @staticmethod
+    def determine_doc_number(reg_str, title, title_part):
+        """Instead of requiring the user provide a doc number, we can find it
```

Comment:
We should probably clarify here (at least in the comments) that this is a Federal Register Document Number -- which is a reasonably specific thing.